armada_jupyter/__main__.py

```diff
@@ -30,6 +30,16 @@ def submit(file: str):
     submit_worker(file, armada_client)
 
 
+@app.command()
+def cancel(url: str):
+    """
+    Accepts a URL and cancels the corresponding job
```

Comment:
The docs don't make it really clear what URL we expect?  I guess it must be the lookout url plus armada?